src/App.jsx

```diff
@@ -38,14 +37,14 @@ export default function App() {
             const incorrectAnswers = result.incorrect_answers.map(incorrectAnswer => {
               return he.decode(incorrectAnswer)
             })
-            const answerChoices = shuffle([...incorrectAnswers, correctAnswer])
+            const choices = shuffle([...incorrectAnswers, correctAnswer])
             return {
               ...result,
-              question: question,
-              correct_answer: correctAnswer,
-              incorrect_answers: incorrectAnswers,
-              choices: answerChoices,
-              selected_answer: undefined
+              question,
+              correctAnswer,
+              incorrectAnswers,
+              choices,
+              selectedAnswer: undefined
```

Comment:
Here I've changed the property names to `camelCase` so they match the style of the rest of the JavaScript code. The property names that are in the API response are in `snake_case`, and using the snake_case names in the application code could be misleading.